_maps/cyberiad.dm

```diff
@@ -18,9 +18,8 @@ z7 = empty
         #include "map_files\cyberiad\z2.dmm"
         #include "map_files\cyberiad\z3.dmm"
         #include "map_files\cyberiad\z4.dmm"
-        #include "map_files\cyberiad\z5.dmm"
+        #include "map_files\generic\z5.dmm"
         #include "map_files\cyberiad\z6.dmm"
-        #include "map_files\cyberiad\z7.dmm"
```

Comment:
Urf, I'm an idiot. you misunderstood what I meant.

Get rid of cyberiad\z7.dmm the *map*, but leave it in, as a generic.